Doc/library/http.server.rst

```diff
@@ -34,14 +34,16 @@ handler.  Code to create and run the server looks like this::
        httpd.serve_forever()
 
 
-.. class:: HTTPServer(server_address, RequestHandlerClass)
+.. class:: HTTPServer(server_address, RequestHandlerClass, \
+                      bind_and_activate=True)
```

Comment:
I would appreciate that we document this parameter if we do that. And thus, I think it's preferrable we split the PR between the bits that only add the `/` marker and those that add a yet-to-be-documented parameter.

Reply:
I found issue #131885 so I'm going to send adding `*`,`/` for that issue, but is it better to make a separate PR for each library or can I send the changes in one PR

Reply:
Documenting a new parameter should be a separate issue as it's a separate doc PR for a separate problem.

Reply:
Oh ok then for this one it's fine to add the missing one.